Map final branch

JuliaS96 opened the pull request:
## Please, go through these steps before you submit a PR.

- [x ] My Pod Leader knows I'm working on this Pull Request
- [x ] I've explained what the Pull Request is adding.
- [x ] I've explained why this is important.

grace-omotoso reviewed (commented):
Remove gemfile changes or you can make me a contributor on your remote branch so I can fix it locally

JuliaS96 commented:
Sorry @grace-omotoso I removed the lockfile but not sure how to delete Gemfile without deleting it for the main so I added you as a collaborator just now!
